### ISSUE TITLE
Implement Datagrid sort feature inside ReferenceManyField

### ISSUE THREAD

**Issue by cytomich** [marmelab/admin-on-rest#364]
Steps to reproduce:

- `make run` in project dir
- go to http://localhost:8080/#/posts/1
- switch to "comments" tab
- click any field in comments grid header

[GIF with bug](http://g.recordit.co/rocSoXzNwv.gif)

```
Uncaught TypeError: _this.props.setSort is not a function
    at Object.Datagrid._this.updateSort [as onClick] (webpack:///../src/mui/list/Datagrid.js?:119:25)
    at EnhancedButton._this.handleClick (webpack:///../~/material-ui/internal/EnhancedButton.js?:142:21)
    at Object.ReactErrorUtils.invokeGuardedCallback (webpack:///../~/react-dom/lib/ReactErrorUtils.js?:70:16)
    at executeDispatch (webpack:///../~/react-dom/lib/EventPluginUtils.js?:85:21)
    at Object.executeDispatchesInOrder (webpack:///../~/react-dom/lib/EventPluginUtils.js?:105:7)
    at executeDispatchesAndRelease (webpack:///../~/react-dom/lib/EventPluginHub.js?:43:22)
    at executeDispatchesAndReleaseTopLevel (webpack:///../~/react-dom/lib/EventPluginHub.js?:54:10)
    at Array.forEach (native)
    at forEachAccumulated (webpack:///../~/react-dom/lib/forEachAccumulated.js?:24:9)
    at Object.processEventQueue (webpack:///../~/react-dom/lib/EventPluginHub.js?:257:7)
```

**Comment by fzaninotto** [comment]
You're right, it's not implemented yet.

**Comment by wesley6j** [comment]
Made a PR: #752

**Comment by fzaninotto** [comment]
Implemented in #752
